src/commands/synthetics/__tests__/run-test.test.ts

```diff
@@ -278,14 +271,19 @@ describe('run-test', () => {
       expect(runTests.getDatadogHost(true, config)).toBe('https://intake.synthetics.datadoghq.com/api/v1')
     })
 
-    test('should be tunable through DATADOG_SITE variable', async () => {
-      process.env = {DATADOG_SITE: 'datadoghq.eu'}
+    test('should use DD_API_HOST_OVERRIDE', async () => {
+      process.env = {DD_API_HOST_OVERRIDE: 'foobar'}
```

Comment:
Nit, to keep the returned result a valid url.
```suggestion
      process.env = {DD_API_HOST_OVERRIDE: 'https://foobar'}
```

Reply:
Done !